From 7961f1f4ac2cce973ecc7a91fa7afd4606d7c7e9 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Tue, 25 Mar 2014 12:16:47 +0800
Subject: [PATCH 71/71] arm: omap2: export prm_read_reset_sources for omap_wdt
 module

This patch is to get omap_wdt compiled as module for watchdog usage.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/mach-omap2/prm_common.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-omap2/prm_common.c b/arch/arm/mach-omap2/prm_common.c
index 228b850..5e9e147 100644
--- a/arch/arm/mach-omap2/prm_common.c
+++ b/arch/arm/mach-omap2/prm_common.c
@@ -21,6 +21,7 @@
 #include <linux/init.h>
 #include <linux/io.h>
 #include <linux/irq.h>
+#include <linux/module.h>
 #include <linux/interrupt.h>
 #include <linux/slab.h>
 
@@ -363,6 +364,7 @@ u32 prm_read_reset_sources(void)
 
 	return ret;
 }
+EXPORT_SYMBOL(prm_read_reset_sources);
 
 /**
  * prm_was_any_context_lost_old - was device context lost? (old API)
-- 
1.7.5.4

